src/main/kotlin/io/emeraldpay/dshackle/rpc/NativeCall.kt

```diff
@@ -378,7 +381,11 @@ open class NativeCall(
 
         companion object {
             const val quoteCode = '"'.code.toByte()
-            val createFilterMethods = listOf("eth_getFilterChanges", "eth_newFilter", "eth_newBlockFilter")
+            val createFilterMethods = listOf(
+                "eth_getFilterChanges",
```

Comment:
looks like mistake here: we have a eth_getFilterChanges method which doesn't create a filter but missed eth_newPendingTransactionFilter which is creates

Reply:
you are right, i'll fix it